http-client/src/main/java/io/airlift/http/client/jetty/JettyHttpClient.java

```diff
@@ -299,6 +303,13 @@ protected void configure(SelectableChannel selectable)
 
         httpClient.setConnectBlocking(config.isConnectBlocking());
 
+        if (config.isExposeJettyMBeans() && mbeanServer != null) {
```

Comment:
Should it be an error, if you asked for mbeans to be enabled and the server is not bound?

Reply:
Makes sense!